doc/lsp.txt

```diff
@@ -277,6 +277,8 @@ keepFocusInReferences	Focus on the location list window after
 noDiagHoverOnLine	Suppress diagnostic hover from appearing when
 			the mouse is over the line instead of the signature.
 			By default this is set to true.
+diagLineHL		Highlight used for diagnostic line.
+			Use `NONE` to disable.
```

Comment:
It'd be nice if this also mentioned what the default is

Reply:
Sure. I'll update.